pycon-us-2018/videos/lightning-talks-saturday-evening-pycon-2018.json

```diff
@@ -0,0 +1,35 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "",
```

Comment:
I just compiled this while watching the video on Youtube:
```
.. list-table:: Lightning Talks
   :widths: 10 30 60
   :header-rows: 1
   * - Time
     - Speaker
     - Title
   * - 00:10_
     - Caleb Collins-Parks
     - AREPL - A real-time python evaluator
   * - 04:35_
     - Michael Foord
     - Abstractions
   * - 10:27_
     - Turicas (Álvaro) Justen
     - Roasting Coffee with Python
   * - 16:27_
     - Petr Viktorin
     - Blinky's Async Adventure 
   * - 22:17_
     - 
     - Type Hint 
   * - 26:25_
     - Scott Sanderson
     - A Class is a Poor Man's...Loop 
   * - 31:25_
     - Nick Humrich
     - RIP HTTP 
   * - 36:08_
     - Allen Downey
     - Modeling and Simulation in Python 
   * - 40:00_
     - Petr Mitev
     - Python in Arch 
   * - 44:53_
     - Steven C. Howell
     - Bayesian Hacking 
   * - 50:47_
     - James Bennett Saxon
     - Code Lives & Things can happen 
   * - 56:42_
     - Andrés García García
     - Why should we care about Processing 
   * - 1:01:40_
     - Meredydd Luff
     - Making the web more pythonic 
   * - 1:07:10_
     - Kenneth Reitz
     - Running for the board﻿

.. _00:10: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=00m10s
.. _04:35: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=04m35s
.. _10:27: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=10m27s
.. _16:27: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=16m27s
.. _22:17: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=22m17s
.. _26:25: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=26m25s
.. _31:25: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=31m25s
.. _36:08: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=36m08s
.. _40:00: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=40m00s
.. _44:53: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=44m53s
.. _50:47: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=50m47s
.. _56:42: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=56m42s
.. _1:01:40: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=61:40
.. _1:07:10: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=67:10
```

Reply:
Great! :+1: 

I will add the `related_urls` when I see this lightning.

This pull request have :ballot_box_with_check: Allow edits from maintainers. If you want, you can add more changes to this pull request. It should be something as adding my repo as remote, modifying it in `Daniel-at-github:pycon-us-2018` branch.